src/modules/home/home.tsx

```diff
@@ -1,191 +1,475 @@
-import './home.scss';
-import { useCallback } from 'react';
-import { useHistory } from 'react-router-dom';
+import "./home.scss";
+import { useCallback } from "react";
+import { useHistory } from "react-router-dom";
+
+function handleAccordionClick(headerId, contentId){
+  var accordionTopButton = document.getElementById(headerId);
+  var accordionContent = document.getElementById(contentId);
+  if(accordionTopButton && accordionContent){
+    accordionTopButton.setAttribute("aria-expanded", "false");
+    accordionContent.setAttribute("hidden", "true");
+  }
+}
+
```

Comment:
Please convert this function to an arrow function.

Reply:
done.  Here and in confirmation.tsx.  Please let me know if it doesn't look right.